### DISCUSSION TITLE
2007/06/14/%E5%8F%88%E6%9C%89%E5%A5%BD%E4%BA%9B%E5%A4%A9%E6%B2%A1%E5%86%99blog%E4%BA%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#164] [Announcements]
# 2007/06/14/%E5%8F%88%E6%9C%89%E5%A5%BD%E4%BA%9B%E5%A4%A9%E6%B2%A1%E5%86%99blog%E4%BA%86

我已经回到上海了。一回来，就发现找不出时间写blog了。上班的时间，要忙着工作，是不能用来写blog的。而下班后，因为交通不便，的赶紧回家。到了家里，没有网络，也写不了。

https://blog.qizhen.xyz/2007/06/14/%E5%8F%88%E6%9C%89%E5%A5%BD%E4%BA%9B%E5%A4%A9%E6%B2%A1%E5%86%99blog%E4%BA%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

yoyo
2007年06月17日 17:07 编辑
这么远 赶紧买车吧

回复
竹贤
2007年06月15日 08:10 编辑
这就是有得有失啊。有了家庭，就有了家庭的幸福，那肯定就得放弃一些个人的空间。

回复
rebecca
2007年06月14日 18:05 编辑
您好,我在网上搜索面试的文章,看到了您写于2005年的博客,您的空间名是"ruanqizhen",真巧我的名字是"ruanqi",一时间我以为自己搞错了,真是巧得很!
